docker-compose.yml

```diff
@@ -4,21 +4,23 @@ networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: niklafinskiy/todoapp:2.1.0
     build:
```

Comment:
The `build` section is specified along with the `image` tag. Typically, you should use either `build` or `image`, but not both. If you want to build the image locally, remove the `image` line.